crates/store/re_protos/src/v1alpha1/rerun.cloud.v1alpha1.ext.rs

```diff
@@ -1178,51 +1182,190 @@ pub struct RegisterWithDatasetTaskDescriptor {
 
 impl ScanPartitionTableResponse {
     pub const PARTITION_ID: &str = "rerun_partition_id";
-    pub const PARTITION_TYPE: &str = "rerun_partition_type";
+
+    /// Layer names for this partition, one per layer.
+    ///
+    /// Should have the same length as [`Self::STORAGE_URLS`].
+    pub const LAYER_NAMES: &str = "rerun_layer_names";
+
+    /// Storage URLs for this partition, one per layer.
+    ///
+    /// Should have the same length as [`Self::LAYER_NAMES`].
+    pub const STORAGE_URLS: &str = "rerun_storage_urls";
+    pub const LAST_UPDATED_AT: &str = "rerun_last_updated_at";
+
+    /// Total number of chunks for this partition.
+    pub const NUM_CHUNKS: &str = "rerun_num_chunks";
+
+    /// Total size in bytes for this partition.
+    pub const SIZE_BYTES: &str = "rerun_size_bytes";
+
+    pub fn layer_names_inner_field() -> FieldRef {
+        Arc::new(Field::new(Self::LAYER_NAMES, DataType::Utf8, false))
+    }
+
+    pub fn storage_urls_inner_field() -> FieldRef {
+        Arc::new(Field::new(Self::STORAGE_URLS, DataType::Utf8, false))
+    }
```

Comment:
I've been trying (and failing) to make sure that all `Field` defining functions are named as `fn field_xxx` so they can easily be lexically grepped for. :shrug: 
<img width="699" height="699" alt="image" src="https://github.com/user-attachments/assets/227f1571-f047-4929-b616-a6db68a57fa0" />
```suggestion
    pub fn field_layer_names_inner() -> FieldRef {
        Arc::new(Field::new(Self::LAYER_NAMES, DataType::Utf8, false))
    }

    pub fn field_storage_urls_inner() -> FieldRef {
        Arc::new(Field::new(Self::STORAGE_URLS, DataType::Utf8, false))
    }
```

Reply:
makes sense